.github/workflows/ci.yaml

```diff
@@ -52,3 +52,45 @@ jobs:
       - uses: actions/checkout@v1
       - run: make install
       - run: make zng-output-check
+  ecr-image-push:
```

Comment:
@marktwallace : You'd asked me at some point during development of this work whether this should be a separate workflow or not, and I told you it could stay in, and I now realize I was wrong: As is, this work will get triggered on for every pull request created, which we don't want. 

Would you please break this out to a separate file in `.github/workflows`, and have the `on:` section be just:
```yaml
  push:
    branches:
      - master
```

That will ensure it only runs when new code lands on master.

Reply:
OK, I will split that out into another workflow -- switching this back to a draft until it is tested

Reply:
I added a new workflow, Deploy, in cd.yaml